vue-frontend/src/App.vue

```diff
@@ -9,12 +12,18 @@ import {
   faArrowLeft,
   faStar,
 } from "@fortawesome/free-solid-svg-icons";
+import Config from "@/config.js";
 
 library.add(faArrowRight, faStar, faArrowLeft);
 
 export default {
   name: "App",
   components: {},
+  data() {
+    return {
+      isNotCrawl: Config.BASE_URL.includes("dev-stack") > -1,
```

Comment:
Instead of checking for string "dev-stack", which we can change anytime, we should add a boolean flag IS_PROD or something that will be true only in prod.